docs/getting-started/install/debian.md

```diff
@@ -18,6 +18,8 @@ sudo apt-get install dokku
 sudo dokku plugin:install-dependencies --core
 ```
 
+These steps are not guaranteed to work on a non-Ubuntu platform. In particular, you'll need to install by hand the sinatra gem on Debian if you want to use the web admin for configuration.
+
```

Comment:
Yeah, but these are not relevant for the *Ubuntu* unattended release, so I'll just adde them in a different code block. I think @josegonzalez suggestion of putting the gem install out of the package management is best the best long term solution?